bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -46,23 +46,39 @@
 		<item-type>Switch</item-type>
 		<label>Night Light</label>
 		<category>Lightbulb</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Light</tag>
+		</tags>
 	</channel-type>
 	<channel-type id="light">
 		<item-type>String</item-type>
 		<label>Illumination</label>
 		<description>Light conditions</description>
+		<tags>
+			<tag>Status</tag>
```

Comment:
```suggestion
			<tag>Measurement</tag>
```

Reply:
ditto

Reply:
The othe rone got adjusted, but not this one. Let's keep it as it is.